share/man/man1/brew.1

```diff
@@ -554,10 +554,6 @@ HOMEBREW_BROWSER
 If set, uses this setting as the browser when opening project homepages, instead of the OS default browser\.
 .
 .TP
-HOMEBREW_BUILD_FROM_SOURCE
```

Comment:
> With reproducible builds: it's as perfect as sha256 is.

Reproducible builds are not perfect for at least two reasons. (a) no all of bottles can be reproducible. (b) it's not resistance to Ken Thomposn Hack, aka, compiler backdoor. (unless you can make build reproducible cross compiler). Apologies on over sensitive on security related issue. Part of the reason is I study cryptography and security as my PhD research field. 

> Allow customisation of this sort makes software worse. Look through our issue tracker and tell me honestly if all users were using binaries we would have more issues rather than less. I don't mean to pull rank but having worked on Homebrew for a long time now: binary packages have solved more support issues than any other individual feature.

This doesn't contradict to my statement. On the contrary is consistent to my logic. That we should support marjor need as default, but keep custom behavior friendly which will only apply to minority people and come with lower support.